### PR TITLE
Added chat

### PR DESCRIPTION
**Pull request by piotrszelejewski**
Yeah, chat

### CODE REVIEW

File: people/templates/people/chat.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  {{ chatMessage.sender.first_name }}
+  {{ chatMessage.sender.last_name }}
+  {{ chatMessage.created_at }}
+</small></p>
+{% endif %}
+{% endfor %}
+
+<form method="post">{%csrf_token%}
+  {{form.as_p}}
+  <input type="submit" value="Send" />
```
Comment (joshdavies14) [inline people/templates/people/chat.html]:
```suggestion
  <input class="btn btn-primary" type="submit" value="Send" />
```

File: people/templates/people/chat.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+{% if user != mentor and user != mentee %}
+<p> You have no permission to acces this page </p>
+{% else %}
+
+{% for chatMessage in chatMessages %}
+
+{% if  chatMessage.sender  == mentor %}
+  {% if  chatMessage.sender  == user %}
+    <p style="text-align:left;background-color:rgb(127, 108, 238);">{{ chatMessage.content }}</p>
```
Comment (joshdavies14) [inline people/templates/people/chat.html]: We probably want to change the colours, or remove them entirely - they aren't the most readable and I imagine wouldn't pass accessibility contrast checks in their current state

**Review by joshdavies14 — CHANGES_REQUESTED** [review]
I think this all LGTM besides my two comments! :)

File: people/templates/people/chat.html (hunk 2 above)
Comment (piotrszelejewski) [reply people/templates/people/chat.html]: I agree

**Review by joshdavies14 — APPROVED** [review]
Sorry. it's been a day. LGTM! :)

**Pull request merged**